bin/pypit_compare_sky

```diff
@@ -8,7 +8,8 @@ import pdb
 import sys, os
 
 this_file = os.path.realpath(__file__)
-sky_path = this_file[:this_file.rfind('/')]+'/../data/sky_spec/'
+#sky_path = this_file[:this_file.rfind('/')]+'/../data/sky_spec/'
```

Comment:
Am sure this is better than what it replaces
but it is still a bit kludgy.  Am ok to run with
it for now..